tests/unit/ReferenceListTest.php

```diff
@@ -372,14 +372,4 @@ public function testGivenNonEmptyList_isNotEmpty() {
 		$this->assertFalse( $references->isEmpty() );
 	}
 
-	public function testGivenNonEmptyListWithForwardedIterator_isNotEmpty() {
-		$references = new ReferenceList();
-		$references->addNewReference( new PropertyNoValueSnak( 1 ) );
-		$references->next();
```

Comment:
Oh, wow. Is this an other line missing in the release notes? "ReferenceList does not implement Iterator any more". Including "ReferenceList does not implement current, next, key, valid and rewind any more" to make it absolutely clear what this means.